script/misc/readme/example-custom-object-formatter.php

```diff
@@ -1,85 +1,68 @@
 <?php
 
-declare(strict_types = 1); // README.md.remove
+declare(strict_types=1);
 
-use Eboreum\Caster\Abstraction\Formatter\AbstractFormatter;
 use Eboreum\Caster\Abstraction\Formatter\AbstractObjectFormatter;
 use Eboreum\Caster\Caster;
-use Eboreum\Caster\CharacterEncoding;
 use Eboreum\Caster\Collection\Formatter\ObjectFormatterCollection;
 use Eboreum\Caster\Contract\CasterInterface;
-use Eboreum\Caster\Formatter\DefaultObjectFormatter;
-use PHPUnit\Framework\MockObject\Generator;
-use PHPUnit\Framework\TestCase;
 
-require_once dirname(__DIR__, 2) . "/bootstrap.php"; // README.md.remove
+require_once dirname(__DIR__, 2) . '/bootstrap.php'; // README.md.remove
 
 $caster = Caster::create();
 
 $caster = $caster->withCustomObjectFormatterCollection(new ObjectFormatterCollection([
     new class extends AbstractObjectFormatter
     {
-        /**
-         * {@inheritDoc}
-         */
         public function format(CasterInterface $caster, object $object): ?string
         {
             if (false === $this->isHandling($object)) {
                 return null; // Pass on to next formatter or lastly DefaultObjectFormatter
             }
 
-            assert($object instanceof \DateTimeInterface);
+            assert($object instanceof DateTimeInterface);
 
             return sprintf(
-                "%s (%s)",
-                Caster::makeNormalizedClassName(new \ReflectionObject($object)),
-                $object->format("c"),
+                '%s (%s)',
+                Caster::makeNormalizedClassName(new ReflectionObject($object)),
+                $object->format('c'),
             );
         }
 
-        /**
-         * {@inheritDoc}
-         */
         public function isHandling(object $object): bool
         {
-            return ($object instanceof \DateTimeInterface);
+            return ($object instanceof DateTimeInterface);
```

Comment:
Syns ikke jeg kan se en use længere oppe?

Reply:
The file isn't in a namespace (it's an example the README.md). As such, imports occur at root level, making the backslash obsolete.